### PR TITLE
Initial changes to modify cluster for use with CYI

### PR DESCRIPTION
**Pull request by connoravo**
Signed-off-by: Connor Avery <ConnorAvery@digital.uc.dwp.gov.uk>

### CODE REVIEW

File: cluster_config.tf [hunk 1]
```diff
@@ -37,9 +32,7 @@ resource "aws_s3_bucket_object" "instances" {
       add_master_sg      = aws_security_group.aws_cyi_infrastructure_common.id
       add_slave_sg       = aws_security_group.aws_cyi_infrastructure_common.id
       subnet_id = (
-        local.use_capacity_reservation[local.environment] == true ?
-        data.terraform_remote_state.internal_compute.outputs.aws_cyi_infrastructure_subnet.subnets[index(data.terraform_remote_state.internal_compute.outputs.aws_cyi_infrastructure_subnet.subnets.*.availability_zone, data.terraform_remote_state.common.outputs.ec2_capacity_reservations.emr_m5_16_x_large_2a.availability_zone)].id :
-        data.terraform_remote_state.internal_compute.outputs.aws_cyi_infrastructure_subnet.subnets[index(data.terraform_remote_state.internal_compute.outputs.aws_cyi_infrastructure_subnet.subnets.*.availability_zone, local.emr_subnet_non_capacity_reserved_environments)].id
+        data.terraform_remote_state.internal_compute.outputs.cyi_subnet.subnets[index(data.terraform_remote_state.internal_compute.outputs.cyi_subnet.subnets.*.availability_zone, local.emr_subnet_region)].id
```
Comment (connoravo) [inline cluster_config.tf:35]: No using capacity reservation - Setting single subnet value via locals

File: bootstrap_actions/download_scripts.sh [hunk 2]
```diff
@@ -27,7 +27,7 @@ chmod u+x /opt/emr/logging.sh
     source /opt/emr/logging.sh
 
     function log_wrapper_message() {
-        log_aws_cyi_infrastructure_message "$${1}" "download_scripts.sh" "$${PID}" "$${@:2}" "Running as: ,$USER"
+        log_cyi_message "$${1}" "download_scripts.sh" "$${PID}" "$${@:2}" "Running as: ,$USER"
```
Comment (steveburton4) [inline bootstrap_actions/download_scripts.sh:30]: Need to change log file location on this one

File: bootstrap_actions/status_metrics.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@
     source /opt/emr/logging.sh
 
     function log_wrapper_message() {
-        log_aws_cyi_infrastructure_message "$${1}" "status_metrics.sh" "$${PID}" "$${@:2}" "Running as: ,$USER"
+        log_cyi_message "$${1}" "status_metrics.sh" "$${PID}" "$${@:2}" "Running as: ,$USER"
     }
```
Comment (steveburton4) [inline bootstrap_actions/status_metrics.sh:14]: Log file location

File: bootstrap_actions/metrics-setup.sh [hunk 4]
```diff
@@ -5,7 +5,7 @@ set -euo pipefail
     source /opt/emr/logging.sh
 
     function log_wrapper_message() {
-        log_aws_cyi_infrastructure_message "$1" "metrics-setup.sh" "$$" "Running as: $USER"
+        log_cyi_message "$1" "metrics-setup.sh" "$$" "Running as: $USER"
     }
```
Comment (steveburton4) [inline bootstrap_actions/metrics-setup.sh:9]: Log file location

File: bootstrap_actions/update_dynamo.sh [hunk 5]
```diff
@@ -42,7 +42,7 @@
   S3_PREFIX=$(cat $S3_PREFIX_FILE)
   SNAPSHOT_TYPE=$(cat $SNAPSHOT_TYPE_FILE)
   EXPORT_DATE=$(cat $EXPORT_DATE_FILE)
-  DATA_PRODUCT="aws_cyi_infrastructure"
+  DATA_PRODUCT="CYI"
 
   if [[ -z "$EXPORT_DATE" ]]; then
```
Comment (steveburton4) [inline bootstrap_actions/update_dynamo.sh:47]: Log file location

File: steps/create-cyi-databases.sh [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    echo "$S3_PREFIX" >>          /opt/emr/s3_prefix.txt
+    echo "$EXPORT_DATE" >>        /opt/emr/export_date.txt
+
+    log_wrapper_message "Creating cyi Databases"
+
+    hive -e "CREATE DATABASE IF NOT EXISTS ${cyi_db} LOCATION '${published_bucket}/${hive_metastore_location}';"
+
+    log_wrapper_message "Finished creating cyi Databases"
+
+) >> /var/log/aws-cyi/create-cyi-databases.log 2>&1
```
Comment (steveburton4) [inline steps/create-cyi-databases.sh:29]:
```suggestion
) >> /var/log/cyi/create-cyi-databases.log 2>&1
```

**Review by steveburton4 — DISMISSED** [review]

**Review by steveburton4 — APPROVED** [review]

**Pull request merged**